test/functorch/test_vmap.py

```diff
@@ -3850,6 +3850,64 @@ def test_threshold(self, device):
         x = torch.randn(2, 3, device=device, requires_grad=True)
         self._batched_grad_test(lambda x: F.threshold(x, 0.5, 0.0), (x,))
 
+    def test_sdpa(self, device):
+        if device == "cpu":
+            raise unittest.SkipTest("This test is only for CUDA for now")
+        from torch.nn.attention import sdpa_kernel, SDPBackend
+
+        backends = [
+            # SDPBackend.MATH,
+            # SDPBackend.EFFICIENT_ATTENTION,
+            SDPBackend.FLASH_ATTENTION  # not supported yet
```

Comment:
are the ones that are commented out not supported? or is FLASH_ATTENTION not supported?

Reply:
oops